clients/web/src/views/AssessmentScorecard.vue

```diff
@@ -41,59 +47,64 @@
 import { Fragment } from 'vue-fragment'
 import API from '@/common/api'
 
-import ButtonPrimary from '@/components/ButtonPrimary'
-import HeaderSecondary from '@/components/HeaderSecondary'
+// import ButtonPrimary from '@/components/ButtonPrimary'
+// import HeaderSecondary from '@/components/HeaderSecondary'
 import LoadingSpinner from '@/components/LoadingSpinner'
 import ScorecardScores from '@/components/ScorecardScores'
-import ScorecardBusinessAreas from '@/components/ScorecardBusinessAreas'
-import ScorecardPractices from '@/components/ScorecardPractices'
-import ScorecardPracticesChart from '@/components/ScorecardPracticesChart'
-import ScorecardTargets from '@/components/ScorecardTargets'
+// import ScorecardBusinessAreas from '@/components/ScorecardBusinessAreas'
+// import ScorecardPractices from '@/components/ScorecardPractices'
+// import ScorecardPracticesChart from '@/components/ScorecardPracticesChart'
+// import ScorecardTargets from '@/components/ScorecardTargets'
 
 export default {
   name: 'assessmentScorecard',
 
   props: ['org', 'id'],
 
-  created() {
-    this.fetchData()
-  },
+  // created() {
+  //   this.fetchData()
+  // },
 
-  methods: {
-    async fetchData() {
-      this.loading = true
-      const [organization, assessment, score] = await Promise.all([
-        this.$context.getOrganization(this.org),
-        this.$context.getAssessment(this.org, this.id),
-        API.getBenchmarks(),
-      ])
+  // methods: {
+  //   async fetchData() {
+  //     this.loading = true
+  //     const [organization, assessment, score] = await Promise.all([
+  //       this.$context.getOrganization(this.org),
+  //       this.$context.getAssessment(this.org, this.id),
+  //       API.getBenchmarks(),
+  //     ])
 
-      this.organization = organization
-      this.assessment = assessment
-      this.assessmentScore = score
-      this.loading = false
-    },
-  },
+  //     this.organization = organization
+  //     this.assessment = assessment
+  //     this.assessmentScore = score
+  //     this.loading = false
+  //   },
+  // },
 
   data() {
     return {
       organization: {},
       assessment: {},
-      assessmentScore: {},
+      assessmentScore: {
+        top: 94,
+        own: 56,
+        average: 76,
+        isValid: true,
+      },
       loading: false,
     }
   },
 
   components: {
-    ButtonPrimary,
+    // ButtonPrimary,
     Fragment,
-    HeaderSecondary,
+    // HeaderSecondary,
     LoadingSpinner,
     ScorecardScores,
-    ScorecardBusinessAreas,
-    ScorecardPractices,
-    ScorecardTargets,
-    ScorecardPracticesChart,
+    // ScorecardBusinessAreas,
+    // ScorecardPractices,
+    // ScorecardPracticesChart,
+    // ScorecardTargets,
   },
```

Comment:
There is a lot of commented code in this file. Usually it is best to simply remove code. It is easier later on when using `git diff`.